types/convict-format-with-validator/convict-format-with-validator-tests.ts

```diff
@@ -0,0 +1,22 @@
+import convictValidators = require('convict-format-with-validator');
+
+// $ExpectType string | undefined
+convictValidators.email.name;
+// $ExpectType ((val: any) => void) | undefined
+convictValidators.email.validate;
```

Comment:
Should be a string according to:
https://github.com/validatorjs/validator.js/blob/master/src/lib/isEmail.js

Reply:
It's the type definition from convict
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/convict/index.d.ts

Reply:
Ok, as we agreed, let's keep the same for interface compatibility.